connector/protobuf/src/test/resources/protobuf/basicmessage_noimports.desc

```
@@ -0,0 +1,18 @@
+
```

Where is this used?
Did you mean to another test?

---

@rangadi yes, it is used in the unit test above, how else can we raise the CANNOT_CONSTRUCT_PROTOBUF_DESCRIPTOR exception?

---

Right. I didn't see the second test carefully.